apps/src/templates/projects/ProjectsGallery.tsx

```diff
@@ -0,0 +1,78 @@
+import React, {useCallback, useMemo, useState} from 'react';
+
+import SegmentedButtons from '@cdo/apps/componentLibrary/segmentedButtons';
+import LibraryTable from '@cdo/apps/templates/projects/LibraryTable';
+import PersonalProjectsTable from '@cdo/apps/templates/projects/PersonalProjectsTable';
+import PublicGallery from '@cdo/apps/templates/projects/PublicGallery';
+import i18n from '@cdo/locale';
+
+import {Galleries} from './projectConstants';
+import {selectGallery} from './projectsRedux';
+
+import moduleStyles from './project-gallery.module.scss';
+
+interface ProjectsGalleryProps {
+  limitedGallery: boolean;
+}
+
+const ProjectsGallery: React.FunctionComponent<ProjectsGalleryProps> = ({
+  limitedGallery,
+}) => {
+  const [selectedTab, setSelectedTab] = useState(Galleries.PRIVATE);
+
+  const galleryTabs = useMemo(() => {
+    const tabs = [
```

Comment:
super minor style thing, but just noting that this could get smooshed into just

```
useMemo(() => ([ /* that tabs array */ ] )
```

and spare the declared variable and explicit return.